readme/clipper.md

```diff
@@ -1,9 +1,37 @@
 # Joplin Web Clipper
 
-The Web Clipper is a browser extension that allows you to save web pages and screenshots from your browser. To start using it, open the Joplin desktop application, go to the **Web Clipper Options** and follow the instructions.
+The Web Clipper is a browser extension that allows you to save web pages and screenshots from your browser. Currently, the Web Clipper is available as a browser extension for Google Chrome as well as Mozilla Firefox. To start using it, open the Joplin desktop application, go to the **Web Clipper Options** and follow the instructions.
 
 <img src="https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/WebExtensionScreenshot.png" style="max-width: 50%; border: 1px solid gray;">
 
+# Installation and Setup
+
+- Install the Joplin Web Clipper browser extension for either [Google Chrome](https://chrome.google.com/webstore/detail/joplin-web-clipper/alofnhikmmkdbbbgpnglcpdollgjjfek) or [Mozilla Firefox](https://addons.mozilla.org/en-US/firefox/addon/joplin-web-clipper/).
+- Install the Joplin Desktop application, if you have not already done so.
+- Open the Joplin Desktop app and select the **Tools** tab at the top.
+- Then, click on the **Options** tab from the dropdown menu.
+- Choose the **Web Clipper** tab from the menu on the left side of the window.
+- Press the button that says **Enable Web Clipper Service**.
+
+Enabling the Web Clipper service will enable the browser extension app to communicate with the Joplin app. This would allow you to access all your saved webpages and browser screenshots taken through the Web Clipper along with all your other notes at one place on the Joplin app. In the same location you can also find an authorization token under the **Advanced Options** section. This token is only needed if you want to allow third-party applications to access Joplin.
+
+# Using the Web Clipper extension
+
+To use the Web Clipper, simply click on the extensions tab that is usually located to the right of the address bar. Select the Joplin Web Clipper from the list of extensions. While using the Web Clipper for the first time, it may ask you to complete authorization in order to access your data. To proceed, open the Joplin desktop application where you will see a pop up. Grant permission to Web Clipper by clicking on the **Grant Authorization** button.
+
+Once you have completed the authorization process, you can see the following menu when you click on the Joplin Web Clipper from the extensions tab.
+
+![Screenshot](https://user-images.githubusercontent.com/58488861/189547712-071948f7-6f2a-4454-80cd-8c2b997e7d91.png)
```

Comment:
That needs to be a link to an image in the joplin repo. The path should be `https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/....`